### PR TITLE
Upgrade Sveltekit

### LINKED ISSUE #3152: Upgrade to Sveltekit `341`
- remove session object

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #3152 

- [x] remove session object

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3161"></a>

### CODE REVIEW

**Comment by Siddhant-K-code** [comment]
reviewing right now

File: src/lib/components/articles.svelte [hunk 1]
```diff
@@ -2,13 +2,13 @@
   import type { BlogPost } from "$lib/types/blog-post";
 
   import MoreArticles from "$lib/components/more-articles.svelte";
-  import { session } from "$app/stores";
+  import { page } from "$app/stores";
```
Comment (Siddhant-K-code) [inline src/lib/components/articles.svelte:5]: 💭 **(non-blocking)**: Any particular reason to name it as `page`? can we get a better/descriptive name?

**Review by Siddhant-K-code — APPROVED** [review]
Looks good to me.

One thing we need to double check after merge, is this Header: Login Get-Demo/ Dashboard Session logic. Will rollback if anything looks flaky.

File: src/lib/components/articles.svelte (hunk 1 above)
Comment (Cahllagerfeld) [reply src/lib/components/articles.svelte:5]: That's a sveltekit thing[[1](https://kit.svelte.dev/docs/modules#$app-stores)]. I don't think there's nothing we can change 😅

**Comment by Cahllagerfeld** [comment]
Lets ship it then 🚢

**Pull request merged**